apis/r/R/write_bioc.R

```diff
@@ -21,7 +21,7 @@ write_soma.DataFrame <- function(
     }
   }
   index <- attr(x, which = 'index')
-  x <- as.data.frame(x)
+  x <- suppressWarnings(as.data.frame(x), classes = "deprecatedWarning")
```

Comment:
Naive question:   If the use is flagged as 'deprecated', there is usually an alternative.  Is there here?

Reply:
Not one that we can easily access; the deprecation warning is triggered from code called by `S4Vectors::as.data.frame,DataFrame-method` and is inaccessible to us outside of rewriting the conversion method

Reply:
Weird. Might be worth asking on either r-package-devel or the bioc slack.  I'd rather code around a warning with newly-blessed better code than to muffle it.

(Don't get me wrong: _here and now_ the muffler is right tool.  No criticism implied.  Mostly 'just curious'.)